lte/gateway/c/oai/tasks/mme_app/mme_app_sgsap_location_update.c

```diff
@@ -56,38 +56,50 @@
 #include "nas_messages_types.h"
 #include "s1ap_messages_types.h"
 #include "sgs_messages_types.h"
-
-/********************************************************************************
- **                                                                            **
- ** Name:               mme_app_send_itti_sgsap_ue_activity_ind()              **
- ** Description         Send UE Activity Indication Message to SGS Task        **
- **                                                                            **
- ** Inputs:              Mobile Id                                             **
- **                                                                            **
-********************************************************************************/
-
-void _mme_app_send_itti_sgsap_ue_activity_ind(
-  const char *imsi, const unsigned int imsi_len)
+#include "nas_proc.h"
+
+static uint8_t _get_eps_attach_type(uint8_t emm_attach_type);
```

Comment:
Move definition of _get_eps_attach_type, _mme_app_update_granted_service_for_ue  and _mme_app_compare_tmsi here